jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/DslTestPlan.java

```diff
@@ -89,6 +95,48 @@ public static DslTestPlan fromJmx(String filePath) throws IOException {
     return new JmxTestPlan(tree);
   }
 
+  /**
+   * if true, thread groups will start consecutively.
+   * One at one time in the order indicated in the plan
+   *
+   * @param serializeTGs enabling or disabling this feature (true/false).
+   * @return this instance for fluent API usage.
+   * @since 0.4
+   */
+  public DslTestPlan setSerialized(Boolean serializeTGs) {
+    this.serializeTGs = serializeTGs;
+    return this;
+  }
+
+  /**
+   * If true, it will cause JMeter to record the data returned from
+   * the server for each sample.
+   * If you have selected a file in your test listeners,
+   * this data will be written to file.
+   *
+   * @param funcMode enabling or disabling this mode (true/false).
+   * @return this instance for fluent API usage.
+   * @since 0.4
+   */
+  public DslTestPlan setGlobalFunctionalMode(Boolean funcMode) {
```

Comment:
I even wonder if is actually neecessary in DSL yo have such method, since JMeter requires in any case in jtlWriters to enable XML format for this to work, users can enable saving request and response data with a java variable. Do you think that could be enough, and maybe is not currently necessary to have this method?

Reply:
drop it